java-checks/src/test/java/org/sonar/java/checks/KnownCapacityHashBasedCollectionCheckTest.java

```diff
@@ -20,18 +20,19 @@
 package org.sonar.java.checks;
 
 import org.junit.jupiter.api.Test;
-import org.sonar.java.checks.verifier.CheckVerifier;
+import org.sonar.java.checks.verifier.internal.InternalCheckVerifier;
 
 import static org.sonar.java.checks.verifier.TestUtils.mainCodeSourcesPath;
 
 class KnownCapacityHashBasedCollectionCheckTest {
 
   @Test
   void test() {
-    CheckVerifier.newVerifier()
+    InternalCheckVerifier.newInstance()
```

Comment:
I realize now that it is misleading having 2 different types of verifier, and one of them as a `.withQuickFixes` method, but actually the `CheckVerifier.newVerifier()` that was there before was already checking quickfixes since we introduced the analyzer-commons-api underneath, so you can undo these changes here